backend/src/routes/user.ts

```diff
@@ -1,7 +1,7 @@
 import { PrismaClient } from "@prisma/client/edge";
 import { withAccelerate } from "@prisma/extension-accelerate";
 import { Hono } from "hono";
-import { sign, verify } from "hono/jwt";
+import { jwt, sign, verify } from "hono/jwt";
```

Comment:
'jwt` is an unused import. if this is done intentionally, please share the reasoning. else, remove it.

Reply:
fixed  : verify and jwt both are unused

Reply:
![image](https://github.com/aadeshkulkarni/medium-app/assets/119005879/da554f0c-2d58-44f1-8c4b-097fdcd89f8c)